src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex.cs

```diff
@@ -14,20 +14,20 @@ internal sealed partial class SyntaxTreeIndex : AbstractSyntaxIndex<SyntaxTreeIn
     private readonly LiteralInfo _literalInfo;
     private readonly IdentifierInfo _identifierInfo;
     private readonly ContextInfo _contextInfo;
-    private readonly HashSet<(string alias, string name, int arity)>? _globalAliasInfo;
+    private readonly HashSet<(string alias, string name, int arity, bool isGlobal)>? _aliasInfo;
```

Comment:
both the global and local alias info is useful to store per file.  So we keep them both, with a flag saying which type it is.